kernel/kernel-api/src/test/scala/io/delta/kernel/internal/SnapshotManagerSuite.scala

```diff
@@ -177,7 +181,10 @@ class SnapshotManagerSuite extends AnyFunSuite with MockFileSystemClientUtils {
       numParts: Int = -1,
       startCheckpoint: Optional[java.lang.Long] = Optional.empty(),
       versionToLoad: Optional[java.lang.Long] = Optional.empty(),
-      v2CheckpointSpec: Seq[(Long, Boolean, Int)] = Seq.empty): Unit = {
+      v2CheckpointSpec: Seq[(Long, Boolean, Int)] = Seq.empty,
+      unbackfilledDeltaVersions: Seq[Long] = Seq.empty,
+      tableCommitCoordinatorClientHandlerOpt:
+        Optional[TableCommitCoordinatorClientHandler] = Optional.empty()): Unit = {
```

Comment:
why do you need to pass the `tableCommitCoordianterClientHandlerOpt`? Can we just pass the `unbackfilledDeltaVersions` as `() -> unbackfilledDeltaVersions` and create the mock handler in this method. Just like how we do for the checkpoint v2.